src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -99,4 +103,18 @@ private void copyUrl() {
         url.putString(USERGUIDE_URL);
         clipboard.setContent(url);
     }
+
+    @FXML
+    private void openUrlInBrowser() {
+        if (Desktop.isDesktopSupported()) {
+            try {
+                Desktop.getDesktop().browse(new URI(USERGUIDE_URL));
+            } catch (IOException | URISyntaxException e) {
+                logger.warning("An error occurred when trying to open the URL: "
```

Comment:
Does logger.warning display the error in the GUI's response panel? If it doesn't it might be confusing to the user since they'll just see no response.

Reply:
should I just remove it?

Reply:
Should definitely leave this in. I was more thinking that it would be nice if the user gets a visible response also, but maybe you can just create another issue for that, and merge this first?